docs/docs/glossary.md

```diff
@@ -0,0 +1,10 @@
+# Glossary
```

Comment:
We should create an issue to track follow up items for documentation and have this on the checklist there